app/controllers/remit_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RemitController < ApplicationController
+  def new
+    @event = Event.find_by(id: params[:id])
```

Comment:
i used find by because find will return an error if not found, whereas find_by will return null.